example_new/FactorsMean.Male.csv

```diff
@@ -1,102 +1,102 @@
-86,284.8356136,47.0148245,45.05908112,2914.632417,1742.712199,1.6,46.20224313,149.2653438
-87,283.8162096,46.76287432,44.89596952,2900.261474,1735.714585,1.6,45.98798451,148.8814748
-88,282.8453171,46.51228788,44.73930989,2886.053351,1728.949099,1.6,45.75046011,148.4976058
-89,281.9273103,46.26606267,44.59014225,2872.123311,1722.464374,1.6,45.50901312,148.1137368
-90,281.0667513,46.02726023,44.44950991,2858.601846,1716.310387,1.6,45.25745051,147.7298678
-91,280.2682139,45.79892732,44.31842331,2845.631534,1710.536895,1.6,45.00781577,147.3459987
-92,279.5361287,45.58402279,44.19782991,2833.363311,1705.192039,1.6,44.79695785,146.9621297
-93,278.8746536,45.38535245,44.08859098,2821.952259,1700.321151,1.6,44.57505864,146.5782607
-94,278.2875754,45.20551357,43.99146609,2811.553014,1695.965788,1.6,44.31769802,146.1943917
-95,277.77824,45.04684955,43.90710488,2802.314967,1692.163025,1.6,44.09683555,145.8105227
-96,277.3495116,44.91141476,43.83604537,2794.37742,1688.944961,1.6,43.84691752,145.4266537
-97,277.0037556,44.80094861,43.77871748,2787.864907,1686.33843,1.6,43.65529621,145.0427847
-98,276.7428383,44.71685771,43.73544996,2782.882885,1684.364873,1.6,43.40912483,144.6589156
-99,276.5681388,44.66020431,43.70647896,2779.513995,1683.04031,1.6,43.20050889,144.2750466
-100,276.4805643,44.63169947,43.69195638,2777.815073,1682.375378,1.6,42.89324576,143.8911776
+Age,FoodCarbohydrate,FoodFat,FoodProtein,FoodSodium,EnergyIntake,PhysicalActivity,Weight,Height
+0,235.742631,33.33161337,36.31297365,2330.388632,1388.206939,1.6,3.5,53
+1,236.5885876,33.44970713,36.44175512,2338.632565,1393.168735,1.6,9,74
+2,238.2633026,33.6838173,36.69676616,2354.959203,1402.994631,1.6,12,87
+3,240.7329509,34.0298526,37.07298579,2379.051467,1417.49242,1.6,14,96
+4,243.9481984,34.48183196,37.56308593,2410.44693,1436.381625,1.6,16,103
+5,247.8458189,35.0320642,38.15766578,2448.552992,1459.302517,1.6,17,107.0732388
+6,252.3507393,35.67137564,38.84554872,2492.666053,1485.827533,1.6,19,113.1410019
+7,257.3784272,36.38937634,39.61412902,2541.993837,1515.474612,1.6,22,118.5760379
+8,262.8375182,37.17475391,40.44975424,2595.679907,1547.721875,1.6,25,123.5277123
+9,268.6325746,38.01558296,41.33812714,2652.829315,1582.023054,1.6,28,128.1463808
+10,274.6668596,38.89963807,42.26471118,2712.534315,1617.823026,1.6,31,132.5689091
+11,280.8450179,39.81469824,43.21512335,2773.899091,1654.572849,1.6,35,137.0298558
+12,287.075556,40.74883148,44.17549941,2836.062536,1691.743705,1.6,39,142.1837018
+13,293.2730342,41.69065002,45.13281842,2898.218239,1728.839261,1.6,43,148.3997796
+14,299.3598919,42.62952801,46.07517571,2959.63102,1765.406022,1.6,48,154.4363642
+15,305.2678567,43.55577608,46.99199649,3019.64955,1801.041398,1.6,53,159.007587
+16,310.9389042,44.46076951,47.87418548,3077.714792,1835.399285,1.6,57,162.2503066
+17,316.3257615,45.33702927,48.71421143,3133.364265,1868.193155,1.6,59.5,164.5644458
+18,421.9083783,60.62066096,64.98933463,4182.737028,2493.176801,1.6,61.5,165.9856801

```

Comment:
There seems to be quite a large step here from age 17 to 18. Is that intentional?